externs/es6.js

```diff
@@ -1140,7 +1142,7 @@ BigUint64Array.of = function(var_args) {};
  * @param {number=} opt_byteOffset
  * @param {number=} opt_byteLength
  * @constructor
- * @extends {ArrayBufferView}
+ * @implements {ArrayBufferView}
  * @throws {Error}
  * @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays/DataView
  */
```

Comment:
Looking quickly at the integer_entities (for some related work, to be submitted as a PR soon), I suspect we just need to duplicate the entities found in `TypedArray`, as it will now be an interface, and classes that implement it now must declare the same signature, rather than just inheriting the native impl.

As an example, for these Uint8Array enjtries:
```
Error:Int8Array.java:21: The type Int8Array must implement the inherited abstract method TypedArray.copyWithin(int, int)
Error:Int8Array.java:21: The type Int8Array must implement the inherited abstract method TypedArray.copyWithin(int, int, int)
Error:Int8Array.java:21: The type Int8Array must implement the inherited abstract method TypedArray.set(TypedArray.SetArrayUnionType, int)
Error:Int8Array.java:21: The type Int8Array must implement the inherited abstract method TypedArray.at(int)
Error:Int8Array.java:21: The type Int8Array must implement the inherited abstract method TypedArray.subarray(int)
Error:Int8Array.java:21: The type Int8Array must implement the inherited abstract method TypedArray.subarray(int, int)
Error:Int8Array.java:21: The type Int8Array must implement the inherited abstract method TypedArray.fill(double, int)
Error:Int8Array.java:21: The type Int8Array must implement the inherited abstract method TypedArray.fill(double, int, int)
Error:Int8Array.java:21: The type Int8Array must implement the inherited abstract method TypedArray.slice(int)
Error:Int8Array.java:21: The type Int8Array must implement the inherited abstract method TypedArray.slice(int, int)
Error:Int8Array.java:21: The type Int8Array must implement the inherited abstract method TypedArray.toString_()
Error:Int8Array.java:21: The type Int8Array must implement the inherited abstract method TypedArray.indexOf(double, int)
Error:Int8Array.java:21: The type Int8Array must implement the inherited abstract method TypedArray.includes(double, int)
Error:Int8Array.java:21: The type Int8Array must implement the inherited abstract method TypedArray.lastIndexOf(double, int)
Error:Int8Array.java:21: The type Int8Array must implement the inherited abstract method TypedArray.toLocaleString()
```
I think the following integer entities, will resolve most of the above (sorted by the order of the errors above they will resolve, not alphabetical as expected):
```
elemental2.core.Uint8Array.copyWithin.target
elemental2.core.Uint8Array.copyWithin.start
elemental2.core.Uint8Array.copyWithin.end
elemental2.core.Uint8Array.set.offset
elemental2.core.Uint8Array.at.index
elemental2.core.Uint8Array.subarray.begin
elemental2.core.Uint8Array.subarray.end
elemental2.core.Uint8Array.fill.begin
elemental2.core.Uint8Array.fill.end
elemental2.core.Uint8Array.slice.begin
elemental2.core.Uint8Array.slice.end
elemental2.core.Uint8Array.indexOf
elemental2.core.Uint8Array.indexOf.fromIndex
elemental2.core.Uint8Array.includes.fromIndex
elemental2.core.Uint8Array.lastIndexOf
elemental2.core.Uint8Array.lastIndexOf.fromIndex
```

This leaves the following, which I don't yet understand:
```
Error:Int8Array.java:21: The type Int8Array must implement the inherited abstract method TypedArray.toString_()
Error:Int8Array.java:21: The type Int8Array must implement the inherited abstract method TypedArray.toLocaleString()
```
and this, which I'm not certain will be resolved (but might be, if the union type is inherited correctly):
```
Error:Int8Array.java:21: The type Int8Array must implement the inherited abstract method TypedArray.set(TypedArray.SetArrayUnionType, int)
```